Boundaries/MWCR/MWCR.json

```diff
@@ -0,0 +1,40 @@
+{
+  "type": "Feature",
+  "properties": {
+    "id": "MWCR",
+    "prefix": [
+      "MWCR"
+    ],
+    "name": "Cayman Approach"
+  },
+  "geometry": {
+    "type": "MultiPolygon",
+    "coordinates": [
+      [
+        [
+          [
+            -79.500000, 
+            20.000000
+            
+          ],
+          [
+            -79.500000,
+            19.333861
+          ],
+          [
+            -80.999722,
+            18.667167 
+          ],
+          [
+            -82.083139,
+            18.667139 
+          ],
+          [
+            -81.999722,
+            20.000000 
+          ]
+        ]
```

Comment:
As defined in the GeoJSON spec, the first and last coordinate sets must be the same, forming a closed volume. Please amend your file so that this is the case.